ERCS/erc-chain_specific_addresses.md

```diff
@@ -0,0 +1,235 @@
+---
```

Comment:
```suggestion
---
eip: 7828
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.